lib/ohai/plugins/windows/virtualization.rb

```diff
@@ -44,6 +44,14 @@
       virtualization[:system] = "kvm"
       virtualization[:role] = "guest"
       virtualization[:systems][:kvm] = "guest"
+    when "American Megatrends Inc."
```

Comment:
This is probably going to catch some systems that are also using American Megatrends BIOS and identify them as VMs when they're actually hardware. We need to key off this and something else that is unique to a hyper-v system.